raiden-ts/tests/integration/raiden.spec.ts

```diff
@@ -1479,4 +1479,49 @@ describe('Raiden', () => {
     // subkey is emptied of ETH
     expect((await sub.getBalance()).isZero()).toBe(true);
   });
+
+  test('suggestPartners', async () => {
```

Comment:
This test case is packed with fixtures and preparations. This makes the test itself hard to read and understand what it is actually testing. Although adding new tests for this function is hard, since all the fixtures are not re-usable. Would be amazing if you would consider to improve here. :upside_down_face:

Reply:
Unfortunately, these integration tests are very specific and hard to make generic fixtures for. #2324 tracks a refactoring of them, we could try to improve it there, but specifically here, I hardly can see how it'd be.

Reply:
Well, for example decare such fixed objects like `suggestResponse` and the mock implementations outside of this test case? They are not the test. 80% of the code is unrelated and I have to parse it myself in the head to see what this test is actually doing. with this pattern unit tests become incredibly hard to to maintain.

Reply:
These are integration tests. Defining mock values somewhere else helps unclog the test, but otoh makes it harder to wrap your mind around what the test does and what's expected data for a given endpoint/method, and splitting it is less useful when the data is useful/relevant only for a single test, but I agree, this can/should be improved, but then it's a matter of refactoring the tests and there's that issue for it.